setup.py

```diff
@@ -37,7 +41,8 @@ def extract_version():
                  'Nd vertical interpolation/stratification of atmospheric '
                  'and oceanographic datasets'),
     version=extract_version(),
-    ext_modules=cythonize(extensions),
+    ext_modules=cythonize(extensions, compiler_directives={'linetrace': True,
+                                                           'binding': True}),
```

Comment:
I've included the directive `binding=True` here as it helped with the cython line tracing and increased the coverage result by ~1%

Reply:
Is there a performance implication? https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#compiler-directives is ambiguous on this front.

Reply:
`linetrace` does not have a performance implication
See here: https://cython.readthedocs.io/en/latest/src/tutorial/profiling_tutorial.html#enabling-line-tracing
Speicifcally:

> Note that even if line tracing is enabled via the compiler directive, it is not used by default. As the runtime slowdown can be substantial, it must additionally be compiled in by the C compiler by setting the C macro definition CYTHON_TRACE=1. To include nogil functions in the trace, set CYTHON_TRACE_NOGIL=1 (which implies CYTHON_TRACE=1).

As for `binding` this no clear guidance on this. I could raise this as a question with cython.